fission-core/library/Fission/Internal/Fixture/Bearer/Nested.hs

```diff
@@ -37,7 +37,7 @@ instance Proof.Resolver InTimeBounds where
 instance ServerDID InTimeBounds where
   getServerDID = pure did
     where
-      Right did = eitherDecode "\"did:key:zStEZpzSMtTt9k2vszgvCwF4fLQQSyA15W5AQ4z3AR6Bx4eFJ5crJFbuGxKmbma4\""
+      Right did = eitherDecode "\"did:key:z2DSW536bcWxPGuz7ZMnXdju64pBoWrybTyzTqWYWa7EjsB\""
```

Comment:
Could we move this & the hardcoded DID in `CLI/Internal/ServerDID` to a shared location so that we only have to update it once in future scenarios? Maybe `Authorization/ServerDID`?

Reply:
@dholms this isn't where we get the _actual_ value — that comes from the `env.yaml`. `InTimeBounds` is a test and development monad. This never gets called at production runtime.

Reply:
ah got it got it :ok_hand: